query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/ExplorerQueries.java

```diff
@@ -93,4 +98,46 @@ static QueryRequest buildQuery1() {
     builder.addGroupBy(Expression.newBuilder().setFunction(dateTimeConvert).build());
     return builder.build();
   }
+
+  static QueryRequest buildQuery2() {
+    Builder builder = QueryRequest.newBuilder();
+
+    Expression apiTraceTags =
+        createComplexAttributeExpression("API_TRACE.tags", "span.kind").build();
+    builder.addSelection(apiTraceTags);
+
+    Filter greaterThanOrEqualToFilter =
+        Filter.newBuilder()
+            .setOperator(Operator.GE)
```

Comment:
nit: we don't want to test a query we'd never make and we don't expose support for comparison filters on strings. Should switch to equality.